py3status/modules/dropboxd_status.py

```diff
@@ -30,6 +32,13 @@
 Requires:
     dropbox-cli: command line interface for dropbox
 
+Notes:
+Some distributions offer an option to install dropbox-cli. If you don't see one for your distribution, then you need to download CLI Python script from Dropbox, rename it to dropbox-cli and make dropbox-cli executable and available in your PATH.
```

Comment:
Can you indent this too for the docs and fix long line? The soft limit is 80. It may be nice to add this [link](https://www.dropbox.com/help/desktop-web/linux-commands#commandshttps://www.dropbox.com/help/desktop-web/linux-commands#commands) too.